client/src/myLinksPage/MyLinks.svelte

```diff
@@ -10,7 +10,8 @@
   //Attaching my links API...
   let dataset = [];
   onMount(async () => {
-    dataset = (await fetchMyLinks()).links
+    dataset = (await fetchMyLinks()).links;
```

Comment:
Should be done on be in db, database query.

Reply:
ah ok